pyproj/geod.py

```diff
@@ -273,13 +280,22 @@ def npts(
         lat2: float,
         npts: int,
         radians: bool = False,
+        initial_idx: int = 1,
+        terminus_idx: int = 1,
+        out_lons: Any = None,
```

Comment:
I think it would simplify things (reduce number of tests needed to ensure everything works properly) and be more clear to leave `out_lons`/`out_lats` as options on `inv_intermediate` and not on `npts`. Thoughts?

Reply:
I don't have a strong opinion here.
I would imagine that whoever cares about performance would not use this function anyway because of `list(zip())` and would use `inv_intermediate` instead.
Would you like me to remove them?

Reply:
> Would you like me to remove them?

That would be great. Thanks :+1:

Reply:
Done